app/Resources/views/article-text.html.twig

```diff
@@ -18,6 +18,12 @@
 
         {% block main %}
 
+            {% if viewSelector %}
+
+                <div data-behaviour="FragmentHandler"></div>
```

Comment:
Hmm, might be better to follow the `hypothesis` variable pattern. Though if https://github.com/elifesciences/journal/pull/907/files#r173397577 is correct, maybe the pattern could declare that it's required? Doesn't work with the `Math` behaviour though...